src/mono/wasm/runtime/es6/dotnet.es6.pre.js

```diff
@@ -1,8 +1,8 @@
-const MONO = {}, BINDING = {}, INTERNAL = {};
+const MONO = {}, BINDING = {}, INTERNAL = {}, IMPORTS = {}, EXPORTS = {};
 let ENVIRONMENT_IS_GLOBAL = false;
 if (typeof createDotnetRuntime === "function") {
     Module = { ready: Module.ready };
-    const extension = createDotnetRuntime({ MONO, BINDING, INTERNAL, Module })
+    const extension = createDotnetRuntime({ MONO, BINDING, INTERNAL, IMPORTS, EXPORTS, Module })
```

Comment:
I don't love the names IMPORTS and EXPORTS for these, it feels a little unclear what direction the importing and exporting is flowing. I don't have any immediate suggestions on how to replace them, though...